src/components/Reactions/ReactionsListModal.tsx

```diff
@@ -69,7 +69,7 @@ export function ReactionsListModal({
           )}
         </div>
         {SelectedEmojiComponent && (
-          <div className='emoji str-chat__message-reaction-emoji str-chat__message-reaction-emoji-big'>
+          <div className='emoji str-chat__message-reaction-emoji str-chat__message-reaction-emoji--with-fallback str-chat__message-reaction-emoji-big'>
```

Comment:
We want to provide some additional styles to prevent layout shift when switching from a fallback emoji to a sprite image. Since only React SDK uses image emoji, I added some react-specific class names.